.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
- [ ] hi @bilalrajput09, As the previous reviewer mentioned we need to setup eslint correctly from this project onwards, so what I can see right now is that you have added the eslint.json file correctly, but you have not updated the `linters.yml` file in your `.github/workflows` directory that is why no eslint check is running see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/223199586-e4ad4b28-965e-4343-b12d-52e791245048.png)

please [go to this link and copy the linters.yml](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml) file and update it at your end, this will make the github actions run eslint checks also, and then your project should be complete, make sure the eslint checks are passing just like the other ones